x-pack/plugins/security_solution/public/timelines/components/timeline/properties/helpers.test.tsx

```diff
@@ -124,9 +125,10 @@ describe('Favorite Button', () => {
         </TestProviders>
       );
 
-      wrapper.simulate('click');
-
-      expect(spy).toHaveBeenCalled();
+      waitFor(() => {
+        wrapper.simulate('click');
+        expect(spy).toHaveBeenCalled();
```

Comment:
shouldn't the expect be outside of waitFor() ?

Reply:
https://testing-library.com/docs/dom-testing-library/api-async/#waitfor